src/libraries/Microsoft.NET.WebAssembly.Threading/src/Microsoft.NET.WebAssembly.Threading.proj

```diff
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <PackageDescription>Exposes Threading APIs for WebAssembly projects</PackageDescription>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <IncludeBuildOutput>true</IncludeBuildOutput>
+    <DebugSymbols>false</DebugSymbols>
+    <DebugType>none</DebugType>
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
This property gets set automatically in packaging.targets: https://github.com/dotnet/runtime/blob/main/eng/packaging.targets#L18-L26.

```suggestion
```